hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/client/DefaultCertificateClient.java

```diff
@@ -183,6 +185,17 @@ private synchronized void loadAllCertificates() {
       getLogger().info("CertificateLifetimeMonitor is disabled for {}",
           component);
     }
+    startRootCaRotationPoller();
```

Comment:
SCMCertificateClient doesn't need this poller.  You can move this under line 174 if statement.